tests/contrib/django/test_connection.py

```diff
@@ -30,7 +26,7 @@ def test_connection(self):
 
         span = spans[0]
         eq_(span.name, 'sqlite.query')
-        eq_(span.service, 'defaultdb')
+        eq_(span.service, 'django-defaultdb')
```

Comment:
Great! let me know if you need help on our side to update your PR! I'll be sure it will be merged immediately after this round of changes!

Thanks a lot for contributing!